### PR TITLE
Replace Dvol with Squeeth vol 

### PR DESCRIPTION
**Pull request by DemolaJames**
**Details**

1. Added Squeeth Vol to the pages below
 - Crab Otc Page
 - Auctions page (admin and user page)

### CODE REVIEW

**Comment by DemolaJames** [comment]
Auctions Page
Screenshot 2022-10-20 at 17 59 50

Crab otc 
Screenshot 2022-10-20 at 17 21 59

**Comment by DemolaJames** [comment]
Auctions Admin view 
image

File: components/utilities/HtmlTooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { styled, Tooltip, tooltipClasses, TooltipProps } from '@mui/material'
+
+export const HtmlTooltip = styled(({ className, ...props }: TooltipProps) => (
+  <Tooltip {...props} classes={{ popper: className }} />
+))(({ theme }) => ({
+  [`& .${tooltipClasses.tooltip}`]: {
+    backgroundColor: '#f5f5f9',
```
Comment (KMKoushik) [inline components/utilities/HtmlTooltip.tsx:7]: Can we change this background color to somewhat dark? Just to make it consistent

**Review by KMKoushik — CHANGES_REQUESTED** [review]
LGTM: with small change

File: components/utilities/HtmlTooltip.tsx (hunk 1 above)
Comment (DemolaJames) [reply components/utilities/HtmlTooltip.tsx]: Very true...made changes

File: constants/message.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
 export const KING_CRAB = 'I am the king crab'
 export const MM_CANCEL = 'I authorize to delete the bid'
 export const GENERAL = 'I authorize'
+export const SQUEETH_REF_VOL_MESSAGE = 'Squeeth reference volatility based on deribit options and squeeth replicating portfolio. Squeeth ref vol is calculated/refreshed every 5 minutes...'
```
Comment (alexisgauba) [inline constants/message.ts:4]:
```suggestion
export const SQUEETH_REF_VOL_MESSAGE = 'Squeeth reference volatility based on deribit options and squeeth replicating portfolio. Squeeth ref vol is calculated/refreshed every 5 minutes.'
```

File: container/CrabV2/Auction/Admin/AuctionDetails.tsx [hunk 3]
```diff
@@ -96,13 +101,21 @@ const AuctionDetails: React.FC = () => {
...
-        <Typography variant="numeric" textAlign="center">
-          {ethDvolIndex}%
+       <Box display="flex" flexDirection="column" justifyContent="center">
+        <Typography color="textSecondary" variant="caption">
+          Squeeth Ref Vol
+          <HtmlTooltip
+            title={
+              <Fragment>
+                {SQUEETH_REF_VOL_MESSAGE}
+                <a href={squeethRefVolDocLink} target="_blank"><b>{'Click here learn more'}</b></a>
```
Comment (alexisgauba) [inline container/CrabV2/Auction/Admin/AuctionDetails.tsx:111]:
```suggestion
                <a href={squeethRefVolDocLink} target="_blank"><b>{'Learn more.'}</b></a>
```

File: container/CrabV2/Auction/Auction.tsx [hunk 4]
```diff
@@ -378,9 +384,19 @@ const AuctionHeaderBody: React.FC<{ osqthEstimate?: string; isUpcoming: boolean
       <Box border=".2px solid grey" height="50px" ml={2} mr={2} />
       <Box display="flex" flexDirection="column" justifyContent="center">
         <Typography color="textSecondary" variant="caption">
-          DVOL
+          Squeeth Ref Vol
+          <HtmlTooltip
+            title={
+              <Fragment>
+                {SQUEETH_REF_VOL_MESSAGE}
+                <a href={squeethRefVolDocLink} target="_blank"><b>{'Click here learn more'}</b></a>
```
Comment (alexisgauba) [inline container/CrabV2/Auction/Auction.tsx:392]:
```suggestion
                <a href={squeethRefVolDocLink} target="_blank"><b>{'Learn more.'}</b></a>
```

File: container/CrabV2/CrabOTC/OTCInfo.tsx [hunk 5]
```diff
@@ -51,9 +56,19 @@ export const OTCInfo: React.FC = () => {
       <Box border=".2px solid grey" height="50px" ml={2} mr={2} />
       <Box display="flex" flexDirection="column" justifyContent="center">
         <Typography color="textSecondary" variant="caption">
-          DVOL
+          Squeeth Ref Vol
+          <HtmlTooltip
+            title={
+              <Fragment>
+                {SQUEETH_REF_VOL_MESSAGE}
+                <a href={squeethRefVolDocLink} target="_blank"><b>{'Click here learn more'}</b></a>
```
Comment (alexisgauba) [inline container/CrabV2/CrabOTC/OTCInfo.tsx:64]:
```suggestion
                <a href={squeethRefVolDocLink} target="_blank"><b>{'Learn more.'}</b></a>
```

**Review by alexisgauba — CHANGES_REQUESTED** [review]

**Comment by alexisgauba** [comment]
Also looks like theres a deployment issue so I can't see the preview @DemolaJames, I'd imagine this also has the[ same issue of the value being wrong](https://github.com/opynfinance/squeeth-monorepo/pull/677#issuecomment-1287605000), but can't check

**Pull request merged**